customer-terminate-service/app/src/main/java/se/inera/intyg/cts/application/service/SendPasswordWithSMS.java

```diff
@@ -0,0 +1,64 @@
+package se.inera.intyg.cts.application.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.cts.domain.model.Termination;
+import se.inera.intyg.cts.domain.repository.TerminationRepository;
+import se.inera.intyg.cts.domain.service.SendPassword;
+import se.inera.intyg.cts.infrastructure.integration.SendSMS;
+
+@Service
+public class SendPasswordWithSMS implements SendPassword {
+
+    private static final Logger LOG = LoggerFactory.getLogger(SendPasswordWithSMS.class);
+
+    private static final String SMS_PREFIX = "sms:";
```

Comment:
I think you should move these to the SendSMSImpl, because they are details related to the integration to TellusTalk.

Reply:
Done